secant/Features/App/AppStore.swift

```diff
@@ -393,10 +397,10 @@ extension AppReducer {
     ) throws -> Initializer {
         do {
             let seedBytes = try mnemonic.toSeed(seedPhrase)
-            let viewingKeys = try derivationTool.deriveUnifiedViewingKeysFromSeed(seedBytes, 1)
-            
+            let spendingKey = try derivationTool.deriveSpendingKey(seedBytes, 0)
+            let viewingKey = try derivationTool.deriveUnifiedViewingKeyFromSpendingKey(spendingKey)
```

Comment:
There's a function of spending key that derives a viewing key

Reply:
Changed